### PR TITLE
Isdb 226 exisiting screens

### PR DESCRIPTION
**Pull request by rahulsanjava19**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/edge-case/address/manual/content.ts [hunk 1]
```diff
@@ -1,64 +1,63 @@
...
-};
-
-const languages = {
-  en,
-  cy,
 };
 
 export const generateContent: TranslationFn = content => {
+  const resourceLoader = new ResourceReader();
+  resourceLoader.Loader('manual-address');
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/address/manual/content.ts:29]: create constant.

File: src/main/steps/edge-case/address/manual/content.ts [hunk 2]
```diff
@@ -1,64 +1,63 @@
...
+  const Translations = resourceLoader.getFileContents().translations;
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/address/manual/content.ts:30]: Translations should be in lower case.

File: src/main/steps/edge-case/address/select/content.test.ts [hunk 3]
```diff
@@ -7,14 +8,16 @@ import {
 
 import { generateContent } from './content';
 
+const resourceLoader = new ResourceReader();
+resourceLoader.Loader('select-address');
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/address/select/content.test.ts:12]: please create constant.

File: src/main/steps/edge-case/address/select/content.test.ts [hunk 4]
```diff
@@ -66,7 +69,7 @@ describe('applicant1 > address > select > content', () => {
     const commonContent1 = { language: 'cy', userCase: { applyingWith: 'alone' } } as CommonContent;
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/address/select/content.test.ts:69]: 1) do we need applyingWith?

2) Also create constants for 'en' and 'cy'

const EN = 'en';
const CY = 'cy';

File: src/main/steps/edge-case/address/select/content.test.ts [hunk 5]
```diff
@@ -66,7 +69,7 @@ describe('applicant1 > address > select > content', () => {
...
 
     const generatedContent1 = generateContent(commonContent1);
-    expect(generatedContent1.section).toBe('Applicant (in Welsh)');
+    expect(generatedContent1.section).toBe('Applicant (in welsh)');
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/address/select/content.test.ts:72]: content should be read from translation. it should be changed to "toBe(cyContent.section)"

File: src/main/steps/edge-case/address/select/content.ts [hunk 6]
```diff
@@ -1,61 +1,64 @@
...
-
-const languages = {
-  en,
-  cy,
 };
 
 export const generateContent: TranslationFn = content => {
+  const resourceLoader = new ResourceReader();
+  resourceLoader.Loader('select-address');
+  const Translations = resourceLoader.getFileContents().translations;
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/address/select/content.ts:27]: Translations should be in lower case.

File: src/main/steps/edge-case/date-of-birth/content.test.ts [hunk 7]
```diff
@@ -1,9 +1,29 @@
 /* eslint-disable @typescript-eslint/ban-types, @typescript-eslint/no-explicit-any */
 import { FormContent, FormFields, FormOptions } from '../../../app/form/Form';
+import { ResourceReader } from '../../../modules/resourcereader/ResourceReader';
 import { CommonContent, generatePageContent } from '../../common/common.content';
 
 import { generateContent } from './content';
 
+const resourceLoader = new ResourceReader();
+resourceLoader.Loader('date-of-birth');
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/date-of-birth/content.test.ts:9]: please create constant.

File: src/main/steps/edge-case/date-of-birth/content.test.ts [hunk 8]
```diff
@@ -126,7 +107,7 @@ it('should have dateOfBirth label when language: en and  applyingWith: alone', (
   const commonContent1 = { language: 'en', userCase: { applyingWith: 'alone' } } as CommonContent;
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/date-of-birth/content.test.ts:107]: 1) do we need applyingWith?

2) Also create constants for 'en' and 'cy'

const EN = 'en';
const CY = 'cy';

File: src/main/steps/edge-case/date-of-birth/content.test.ts [hunk 9]
```diff
@@ -78,13 +64,6 @@ describe('appllicant1 > dob-content', () => {
     expect((form.submit.text as Function)(generatePageContent({ language: EN }))).toBe('Save and continue');
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/date-of-birth/content.test.ts:64]: All the toBe content should be read from translation file.

File: src/main/steps/edge-case/date-of-birth/content.ts [hunk 10]
```diff
@@ -1,96 +1,79 @@
...
-  },
 };
 
-const languages = {
-  en,
-  cy,
-};
+export const generateContent: TranslationFn = content => {
+  const resourceLoader = new ResourceReader();
+  resourceLoader.Loader('date-of-birth');
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/date-of-birth/content.ts:49]: please create constant.

File: src/main/steps/edge-case/date-of-birth/content.ts [hunk 11]
```diff
@@ -1,96 +1,79 @@
...
+  const Translations = resourceLoader.getFileContents().translations;
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/date-of-birth/content.ts:50]: Translations should be in lower case.

File: src/main/steps/edge-case/full-name/content.test.ts [hunk 12]
```diff
@@ -70,9 +63,5 @@ describe('Applicant > full-name', () => {
   test('should contain submit button', () => {
     expect((form.submit.text as Function)(generatePageContent({ language: 'en' }))).toBe('Save and continue');
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/full-name/content.test.ts:64]: 1) All the toBe content should be read from translation file.

2) Also create constants for 'en' and 'cy'

const EN = 'en';
const CY = 'cy';

File: src/main/steps/edge-case/full-name/content.ts [hunk 13]
```diff
@@ -1,68 +1,54 @@
...
-};
-
-const languages = {
-  en,
-  cy,
 };
 
 export const generateContent: TranslationFn = content => {
+  const resourceLoader = new ResourceReader();
+  resourceLoader.Loader('full-name');
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/full-name/content.ts:22]: please create constant.

File: src/main/steps/edge-case/full-name/content.ts [hunk 14]
```diff
@@ -1,68 +1,54 @@
...
+  const Translations = resourceLoader.getFileContents().translations;
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/full-name/content.ts:23]: Translations should be in lower case.

File: src/main/steps/edge-case/user-role/content.test.ts [hunk 15]
```diff
@@ -1,27 +1,27 @@
...
 
 import { generateContent } from './content';
 
 jest.mock('../../../app/form/validation');
 
-const en = () => ({
-  continue: 'Save and continue',
-});
+const resourceLoader = new ResourceReader();
+resourceLoader.Loader('user-role');
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/user-role/content.test.ts:10]: please create constant.

File: src/main/steps/edge-case/user-role/content.test.ts [hunk 16]
```diff
@@ -30,26 +30,22 @@ describe('role-type content', () => {
   const commonContent = { language: 'en', userCase: {} } as CommonContent;
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/user-role/content.test.ts:30]: create constants for 'en' and 'cy'

const EN = 'en';
const CY = 'cy';

File: src/main/steps/edge-case/user-role/content.ts [hunk 17]
```diff
@@ -1,63 +1,60 @@
...
 };
 
-const languages = {
-  en,
-  cy,
-};
-
 export const generateContent: TranslationFn = content => {
+  const resourceLoader = new ResourceReader();
+  resourceLoader.Loader('user-role');
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/user-role/content.ts:30]: please create constant.

File: src/main/steps/edge-case/user-role/content.ts [hunk 18]
```diff
@@ -1,63 +1,60 @@
...
+  const Translations = resourceLoader.getFileContents().translations;
```
Comment (parvezhmcts) [inline src/main/steps/edge-case/user-role/content.ts:31]: Translations should be in lower case.

**Review by parvezhmcts — CHANGES_REQUESTED** [review]
Please check review comments.

File: src/main/app/address/AddressLookupPostControllerBase.ts [hunk 19]
```diff
@@ -18,7 +18,8 @@ export default class AddressLookupPostControllerBase extends PostController<AnyO
 
     let addresses;
 
-    const form = new Form(<FormFields>this.fields);
+    const fields = typeof this.fields === 'function' ? this.fields(req.session.userCase) : this.fields;
```
[the remark this answers is not in the document]
Comment (rahulsanjava19) [reply src/main/app/address/AddressLookupPostControllerBase.ts:21]: This code is needed b'coz from content.js file we passed function so need to change this. This was the problem I was talking in stand up

[the remark this answers is not in the document]
Comment (rahulsanjava19) [reply src/main/app/address/AddressLookupPostControllerBase.ts:21]: Removed

File: src/main/steps/edge-case/address/lookup/content.test.ts [hunk 20]
```diff
@@ -8,14 +9,16 @@ import { MANUAL_ADDRESS } from '../../../urls';
 
 import { generateContent } from './content';
 
+const resourceLoader = new ResourceReader();
+resourceLoader.Loader('address-lookup');
```
[the remark this answers is not in the document]
Comment (rahulsanjava19) [reply src/main/steps/edge-case/address/lookup/content.test.ts:13]: done

File: src/main/steps/edge-case/address/lookup/content.ts [hunk 21]
```diff
@@ -1,58 +1,59 @@
...
 };
 
-const languages = {
-  en,
-  cy,
-};
-
 export const generateContent: TranslationFn = content => {
+  const resourceLoader = new ResourceReader();
+  resourceLoader.Loader('address-lookup');
```
[the remark this answers is not in the document]
Comment (rahulsanjava19) [reply src/main/steps/edge-case/address/lookup/content.ts]: done

**Review by parvezhmcts — APPROVED** [review]
looks ok now.
